src/Symfony/Component/HttpKernel/EventListener/SessionListener.php

```diff
@@ -24,6 +25,8 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
+ * To be removed in Symfony 6.0 the logic is now handled by The SessionStorageFactories
+ *
  * @final
  */
 class SessionListener extends AbstractSessionListener
```

Comment:
What happens to logical lines of code located in the abstract class? I couldn't manage to find it.